lib/matplotlib/backend_bases.py

```diff
@@ -3180,9 +3180,9 @@ def release_zoom(self, event):
         for i, ax in enumerate(self._zoom_info.axes):
             # Detect whether this Axes is twinned with an earlier Axes in the
             # list of zoomed Axes, to avoid double zooming.
-            twinx = any(ax.get_shared_x_axes().joined(ax, prev)
+            twinx = any(ax._shared_axes["x"].joined(ax, prev)
```

Comment:
The view is sufficient here. I think we should not drop to the private API if not needed. If performance is a concern, let’s pre-fetch the GrouperView once

```suggestion
            shared_axes_x = ax._shared_axes["x"]
            twinx = any(shared_axes_x.joined(ax, prev)
```

Reply:
sure; I don't think perf is a problem either.